src/test/java/com/hotels/hcommon/hive/metastore/paths/UnpartitionedTablePathResolverTest.java

```diff
@@ -30,21 +30,17 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
 public class UnpartitionedTablePathResolverTest {
 
-  private static final String DATABASE_NAME = "database";
-  private static final String TABLE_NAME = "table";
   private TablePathResolver tablePathResolver;
   private @Mock Table table;
   private @Mock StorageDescriptor tableSd;
 
   @Before
   public void setUp() {
-    when(table.getDbName()).thenReturn(DATABASE_NAME);
```

Comment:
No longer necessary?

Reply:
Nope, new version of Mockito actually fails the test if you stub something but it then never gets called. So it forces you to remove these, which is great.